src/main/java/org/ibp/api/java/germplasm/GermplasmService.java

```diff
@@ -59,4 +63,9 @@
 
 	List<org.generationcp.middleware.api.attribute.AttributeDTO> getGermplasmAttributesByCodes(Set<String> codes);
 
+	List<AttributeDTO> getGermplasmAttributes();
+
+	List<Method> getAllBreedingMethods();
+
+	List<GermplasmName> getGermplasmNames();
```

Comment:
Same here, we could use getGermplasmNameTypesByCodes and rename it to filterGermplasmNameTypes

Reply:
Done.